pipelines/rj_smas__disparo_pic/utils/discord.py

```
@@ -0,0 +1,224 @@
+    if sample_destination:
+        message += f"""
+📱 **Exemplo de Disparo:**
+```json
+{_format_sample_destination(sample_destination)}
+```
+"""
+
+    try:
+        asyncio.run(_send_discord_webhook(webhook_url, message))
+        log("Discord notification sent successfully")
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)